src/utils/storage.ts

```diff
@@ -61,10 +64,36 @@ export const setStorageItem = async <Value extends unknown>({
 
   const valueToSet = typeof value === 'string' ? value : JSON.stringify(value);
 
-  storage.setItem(key, valueToSet);
+  if (isClientSide && typeof valueToSet === 'string') {
+    storage.setItem(key, valueToSet);
+    storage.setItem(`${key}_update_after`, getThisWeek());
+    await mutate(key, valueToSet, false);
+  } else if (isClientSide) {
+    storage.removeItem(key);
+    storage.removeItem(`${key}_update_after`);
+    await mutate(key, null, false);
+  }
```

Comment:
I don't think this code makes sense in this context anymore. This code specifically will always satisfy the first condition (it gets stringified if it is not a string). The else if condition will never run. 

I think this should be removed out of here and be done in the hook instead. Also I realize that we aren't actually using SWR for this amino hook now anyway so the mutate can probably be removed.